google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -302,12 +317,17 @@ def get_columns(self, connection, table_name, schema=None, **kw):
             columns = snap.execute_sql(sql)
 
             for col in columns:
-                type_ = "STRING" if col[1].startswith("STRING") else col[1]
+                if col[1].startswith("STRING"):
+                    end = col[1].index(")")
+                    size = int(col[1][7:end])
+                    type_ = _type_map["STRING"](length=size)
+                else:
+                    type_ = _type_map[col[1]]
```

Comment:
While reflecting column types, we're reading strings like `STRING(size)". Such a string can be parsed to get the columns size - to set it into SQLAlchemy reflected type.